src/main/resources/i18n/Enums_es.properties

```diff
@@ -0,0 +1,141 @@
+# GENERATED Wed, 15 Mar 2023 17:03:17 GMT - DO NOT EDIT
+nursery.BatchQuantityHistoryType.Computed=Computado
+nursery.BatchQuantityHistoryType.Observed=Observado
+nursery.WithdrawalPurpose.Dead=Muerto
+nursery.WithdrawalPurpose.NurseryTransfer=Transferencia de viveros
+nursery.WithdrawalPurpose.Other=Otros
+nursery.WithdrawalPurpose.OutPlant=Planta exterior
+public.DeviceTemplateCategory.PV=PV
+public.DeviceTemplateCategory.SeedBankDefault=Impago del banco de semillas
```

Comment:
I don't remember where we use this, but it could be "Banco de semillas por defecto" instead of "impago"